### ISSUE TITLE
Changing learning rate for BertModelforTokenClassification

### ISSUE THREAD

**Issue by YojanaGadiya** [huggingface/transformers#7264]
Dear all,

I wanted to set a different learning rate for the linear layer and the Bert model for a BertModelforTokenClassification. How can I do so? This change should be done after loading a locally saved BertModelforTokenClassification model.

Thank You.

**Comment by bugface** [comment]
You can specify the learning rate in the optimizer, see the example using SGD below (ref: https://discuss.pytorch.org/t/different-learning-rate-for-a-specific-layer/33670)

```
optim.SGD([
                {'params': model.base.parameters()},
                {'params': model.classifier.parameters(), 'lr': 1e-3}
            ], lr=1e-2)
```
